src/redux/actions/embedding/loadEmbedding.js

```diff
@@ -3,10 +3,12 @@ import { fetchCachedWork } from '../../../utils/cacheRequest';
 
 const TIMEOUT_SECONDS = 90;
 
-const loadEmbedding = (experimentId, embeddingType) => async (dispatch, getState) => {
+const loadEmbedding = (experimentId, embeddingType, forceReload) => async (dispatch, getState) => {
   // If a previous load was initiated, hold off on it until that one is executed.
-  if (getState().embeddings[embeddingType]?.loading
-    || getState().embeddings[embeddingType]?.data.length) {
+  if (!forceReload && (
```

Comment:
Is this `forceReload` being used at all? I can't actually see it anywhere apart from in the tests

Reply:
It's used in this line in `configureEmbedding` to force embedding to reload : https://github.com/biomage-ltd/ui/blob/798a3f8006b856b3f7ed424a7e62708c34f4c662/src/components/data-processing/ConfigureEmbedding/CalculationConfig.jsx#L122

Reply:
I see, this is why my search for `forceReload` returned nothing. Makes sense! Maybe you would want to put a default value for `forceReload`?

Reply:
Hmm.. ok. If it's not passed in, it defaults to `undefined`, which is checked here. But you're right, explicitly setting a default value for it makes it clearer. I'll add it.

Reply:
great!

Reply:
default value `false` added in commit 925427b